source/sync/configure/sync-settings.txt

```diff
@@ -25,23 +23,57 @@ Device Sync <enable-sync>`.
 Available Settings
 ------------------
 
+.. _sync-config-data-source:
+
+Cluster to Sync
+~~~~~~~~~~~~~~~
+
+The name of the linked Atlas cluster :ref:`data source <data-sources>` where
+you want to store your synced data.
+
+You cannot modify this field while Device Sync is enabled. You must
+:ref:`terminate Sync <pause-or-terminate-sync>` before you can select a
+different cluster.
+
+.. include:: /includes/note-flexible-sync-prerequisite.rst
+
 .. _flexible-sync:
 
 Sync Type
 ~~~~~~~~~
 
-Atlas Device Sync has two sync modes: Flexible Sync and the older
-Partition-Based Sync. Partition-Based Sync has 
-been deprecated and is disallowed for new Sync configurations. If you have 
-an existing app that uses Partition-Based Sync, you can migrate 
-to Flexible Sync. For more information, refer to :ref:`realm-sync-migrate-modes`.
+.. important:: Sync Type Displays Only for Partition-Based Sync Apps
```

Comment:
Minor suggestion to avoid "displays" and match the wording in the description.

```suggestion
.. important:: Sync Type is Only Available for Partition-Based Sync Apps
```